react/features/filmstrip/components/web/StatusBar.js

```diff
@@ -0,0 +1,112 @@
+/* @flow */
+
+import React, { Component } from 'react';
+
+import { getLocalParticipant, getParticipantById, PARTICIPANT_ROLE } from '../../../base/participants';
+import { connect } from '../../../base/redux';
+import { getCurrentLayout, LAYOUTS } from '../../../video-layout';
+
+import AudioMutedIndicator from './AudioMutedIndicator';
+import ModeratorIndicator from './ModeratorIndicator';
+import VideoMutedIndicator from './VideoMutedIndicator';
+
+declare var interfaceConfig: Object;
+
+/**
+ * The type of the React {@code Component} props of {@link StatusBar}.
+ */
+type Props = {
+
+    /**
+     * The current layout of the filmstrip.
+     */
+    _currentLayout: string,
+
+    /**
+     * Indicates if the moderator indicator should be visible or not.
+     */
+    _showModeratorIndicator: Boolean,
+
+    /**
+     * Indicates if the audio muted indicator should be visible or not.
+     */
+    showAudioMutedIndicator: Boolean,
+
+    /**
+     * Indicates if the video muted indicator should be visible or not.
+     */
+    showVideoMutedIndicator: Boolean,
+
+    /**
+     * The ID of the participant for which the status bar is rendered.
+     */
+    participantID: String
+};
+
+/**
+ * React {@code Component} for showing the status bar in a thumbnail.
+ *
+ * @extends Component
+ */
+class StatusBar extends Component<Props> {
```

Comment:
Not a fan of this name, this is not a bar. These are the participant's status indicators...

Reply:
I don't have any preferences for the name but I also don't have better ideas for the name. I've used this one because the method that updates it is currently called `updateStatusBar`. Do you have better ideas? I will rename it to whatever you like.

Reply:
How about `StatusIndicators` ?